src/coreclr/src/debug/daccess/request.cpp

```diff
@@ -4065,6 +4067,75 @@ PTR_IUnknown ClrDataAccess::DACGetCOMIPFromCCW(PTR_ComCallWrapper pCCW, int vtab
 }
 #endif
 
+#ifdef FEATURE_COMWRAPPERS
+HRESULT ClrDataAccess::DACTryGetComWrappersObjectFromCCW(CLRDATA_ADDRESS ccwPtr, OBJECTREF* objRef)
+{
+    if (ccwPtr == 0 || objRef == NULL)
+        return E_INVALIDARG;
+
+    SOSDacEnter();
+
+    // Read CCWs QI address and compare it to the managed object wrapper's implementation.
+    ULONG32 bytesRead = 0;
+    CLRDATA_ADDRESS vTableAddress = NULL;
+    IfFailGo(m_pTarget->ReadVirtual(ccwPtr, (PBYTE)&vTableAddress, sizeof(CLRDATA_ADDRESS), &bytesRead));
```

Comment:
I think you wanted a pointer sized read here right? sizeof(CLRDATA_ADDRESS) is 8 regardless whether you are debugging a 32 or 64 bit target. sizeof(TADDR) is probably what you wanted, 4 when debugging 32 bit targets and 8 when debugging 64 bit targets.